extensions-core/druid-bloom-filter/src/main/java/org/apache/druid/query/filter/BloomDimFilter.java

```diff
@@ -123,13 +123,13 @@ public DruidFloatPredicate makeFloatPredicate()
               @Override
               public boolean applyFloat(float input)
               {
-                return bloomKFilter.testFloat(input);
+                return bloomFilter.testDouble(input);
```

Comment:
I especially don't like this change, going to try and find something better to do here.